### PR TITLE
Dependencies file to allow HF Postings-OneBOM automation

### PR DESCRIPTION
**Pull request by jacekpawlak**
### Details:
This file is added in order to be used in automation for HuggingFace Hub IR Models postings with OneBOM dependency checks

### Tickets:
CVS-139271

### CODE REVIEW

File: docs/dev/ov_dependencies_24_2.txt [hunk 1]
```diff
@@ -0,0 +1,374 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
```
Comment (kblaszczak-intel) [inline docs/dev/ov_dependencies_24_2.txt:4]: I would suggest adding a description of the file's content and purpose here, something like:
# this file provides a comprehensive list of all dependencies of .......
# The file is part of the automation pipeline for posting OpenVINO IR models on the HuggingFace Hub, including OneBOM dependency checks.

Comment (jacekpawlak) [reply docs/dev/ov_dependencies_24_2.txt:5]: Added disclaimer

**Review by Dmi3shin — COMMENTED** [review]
LGTM

**Review by akazakov-github — APPROVED** [review]
lgtm

**Pull request merged**